textile-utils-api-v1/build.gradle

```diff
@@ -0,0 +1,10 @@
+archivesBaseName = 'textile-utils-api-v1'
+version = '1.0.0'
+
+dependencies {
+	implementation project(':textile-api-base')
+}
+
+minecraft {
+	accessWidener "src/main/resources/textile-utils-api-v1.accessWidener"
```

Comment:
remove

Reply:
Why? We needed to open up Block.id, Item.id and Block.BLOCKS from being final

Reply:
doesnt need an access widener

Reply:
I’m afraid I still don’t understand what you mean. Do you mean 1) that what is being widened is unnecessary, 2) that although it is needed for compilation it is not needed in production, or 3) something else?

Reply:
it is not necessary to use an access widener here. accessors will suffice

Reply:
Accessors can modify final variables? I had forgotten that. Okay that makes sense then.

Reply:
Yep. Adding a  `@Mutable` does the trick